lib/facter/facts/freebsd/mountpoints.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Facts
+  module Freebsd
+    class Mountpoints
+      FACT_NAME = 'mountpoints'
+
+      def call_the_resolver
+        mountpoints = Facter::Resolvers::Linux::Mountpoints.resolve(FACT_NAME.to_sym)
```

Comment:
Could you please remove the Linux module from the mountpoints resolver's namespace, because now this resolver is used on two platforms.